.bazelversion

```diff
@@ -1 +1 @@
-6.0.0-pre.20220421.3
\ No newline at end of file
+5.4.1
```

Comment:
Did we just have a mismatch in bazel versions?

Reply:
So as far as I can tell the breaking change whereby `@bazel_tools//platforms` was removed occurred between the prerelease that we previously depended upon and `6.0.0`. `5.4.1` still has these tool definitions and the build still works